# This is an action definition file, which has three parts: the goal, the
# result, and the feedback.
#
# Part 1: the goal, to be sent by the client
# The amount of time we want to wait

uint32 time_to_wait

---
# Part 2: the result, to be sent by the server upon completion
# How much time we waited

uint32 time_elapsed

---
# Part 3: the feedback, to be sent periodically by the server during
# execution.
# The amount of time that has elapsed from the start

uint32 time_elapsed

# The amount of time remaining until we're done

uint32 time_remaining
